mmp_to_musicxml/converter.py

```diff
@@ -202,17 +202,23 @@ class MMP_MusicXML_Converter:
 	NOTE_ADJUSTER = None
 	
 	SPECIFIED_KEY_SIGNATURE = None
-	
-	def __init__(self, check_notes=False, key_signature=None):
+
+	opts = None
+
+	def __init__(self, key_signature=None, params=None):
 		logging.basicConfig(level=logging.DEBUG)
-		
-		if check_notes:
+
+		if 'opts' in params: self.opts = params['opts']
```

Comment:
looks great overall! 

just one small adjustment is needed here in case `params` is `None` (and to get my tests to pass 😆):
```
if params and 'opts' in params: self.opts = params['opts']
if params and 'minor' in params: self.minor = params['minor']
```

Reply:
Sure. How can I run thoses tests myself? I'll do that before submitting pull requests from now on.

Reply:
thanks! if you install `pytest`, you can run `pytest` in the root directory to run the tests.

the output should look like:
![image](https://github.com/syncopika/mmp-to-MusicXML/assets/8601582/ba4722c7-fa28-48ae-ac61-a1ae3a476706)